app/admin/dashboard.rb

```diff
@@ -43,7 +43,10 @@ def recupere_support
     end
 
     def recupere_evaluations
-      @evaluations = Evaluation.accessible_by(current_ability).order(created_at: :desc).limit(10)
+      @evaluations = Evaluation.accessible_by(current_ability)
+                               .non_anonymes
+                               .order(created_at: :desc)
+                               .limit(10)
```

Comment:
Ce que j'aime bien faire, c'est extraire la logique métier dans un scope/méthode.

Cela a 2 avantages : 
- On utilise le langage métier
- On peut tester plus facilement les conditions d'affichage

Ce que j'imagine : 

```suggestion
      @evaluations = Evaluation.tableau_de_bord(current_ability)
```

```ruby
def tableau_de_bord(current_ability)
  accessible_by(current_ability)
    .non_anonymes
    .order(created_at: :desc)
    .limit(10)
end
```

qu'est ce que tu en pense ?

Reply:
J'ai même propagé l'idée à un autre endroit du fichier